src/app/features/business/components/dashboard/overview/overview.component.html

```diff
@@ -4,97 +4,124 @@
 <ng-container *ngIf="investorPreparednessGeneralSummary$ | async"></ng-container>
```

Comment:
Here, it would be best to combine these async operations so that we dont overload the template:

<ng-container *ngIf="{
  investorEligibilityGeneralSummary: investorEligibilityGeneralSummary$ | async,
  triggerShowFilterDialog: triggerShowFilterDialog$ | async,
  preparednessSubmissions: preparednessSubmissions$ | async,
  factSheetSubmissions: factSheetSubmissions$ | async,
} as observables"></ng-container>